typescript/cli/src/tests/nodes.ts

```diff
@@ -45,3 +57,68 @@ export async function runCosmosNode({ rpcPort, restPort }: TestChainMetadata) {
 
   return container;
 }
+
+export async function runRadixNode(
```

Comment:
I wonder if this is something we eventually add into an altvm test interface

Reply:
it would probably makes sense to, but we don't have a proper plan for multi vm testing yet. would be cool to have a single test package and run the same test suite against the different vm implementation which i think it is something that also Andrey was mentioning in his doc

Reply:
So with AltVM at least it is only necessary to test the AltVM modules once like the `AltVMWarpModule.test.ts`, but I have not written them for the other modules yet because with MultiVM it will change anyway. But then you only need to test the protocols like radix against their AltVM interface and nothing more so you also save a lot of e2e tests. But like you said I considered it not worth it anymore to write out everything when it will change sooner or later